src/c/helpers.h

```diff
@@ -33,7 +33,11 @@
 #endif
 #endif
 
-
+#ifdef __MINGW32__
+// Mingw does not have an sa_family_t according to:
+//   https://lists.gnu.org/archive/html/bug-gnulib/2009-05/msg00194.html
+typedef unsigned short sa_family_t;
+#endif
 
```

Comment:
Could you restore the big vertical whitespace between the definitions and the callbacks that was there before this change?